openmc/capi/math.py

```diff
@@ -102,7 +113,31 @@ def evaluate_legendre(data, x):
     data_arr = np.array(data, dtype=np.float64)
     return _dll.evaluate_legendre_c(len(data),
                                     data_arr.ctypes.data_as(POINTER(c_double)),
-                                    x)
+                                    x, flag)
+
+
+def calc_norm_pn(l, zmin, zmax):
+    """ Calculate normalization arrays for n orders in Legendre polynomial.
+
+    Parameters
+    ----------
+    l : int
+        Maximum order
+    zmin : float
+        The lowest boundary of the domain
+    zmax : float
+        The highest boundary of the domain
+
+    Returns
+    -------
+    numpy.ndarray
+        Corresponding resulting list of normalization coefficients
+
+    """
+
+    norm_pn = np.empty(l + 1, dtype=np.float64)
+    _dll.calc_norm_pn_c(l, zmin, zmax, norm_pn)
```

Comment:
I don't see a good reason for this function to be implemented in C++ and then exposed to Python. Is there a reason we would need it on the C++ side? If not, I would suggestion creating a new module `openmc/polynomial.py` with polynomial-related functionality (like getting normalization coefficients), move this function there and refactor to not call C++ code. Should be as simple as `return (2.0*np.arange(l) + 1.0)/(domain_max - domain_min)`.